### ISSUE TITLE
Allow tab-completing when the last character before is , not only space

### ISSUE THREAD

**Issue by Mikaela** [weechat/weechat#642]
Example case: I am on `&bitlbee` and want to send message to multiple users.

I do `/msg ro<tab>` which results to `/msg root` and then try to `/msg root,ci<tab>` and nothing happens. I must make space instead of comma and then I can `/msg root ciblia` and then I must go back and change the line to `/msg root,ciblia` and only then I can send the same PM to both.

What I would like is that when I autocomplete root and put comma, I could autocomplete ciblia too like the comma was space.

I hope you understand, but if not, say, and I will try to explain it better, but I am not sure if I can.

* WeeChat 1.4-rc1 (git: v1.4-rc1-1-g3330724) Copyright (C) 2003-2016, compiled on Jan  2 2016 10:49:03
* Debian GNU/Linux 7.9 (wheezy)

**Comment by flashcode** [comment]
I already discussed that on irc some weeks ago, the solution I proposed is not to use comma as another separator, which would be difficult to implement because the space delimits arguments (and completion is different for each comment argument), while the comma is just a char in an argument.
Instead I propose to have a custom completion for commands like `/msg`, ie if you do `/msg #chan1,#c[TAB]`, it would auto split internally on the comma and complete the second channel, keeping first chan as-is.

#weechat log:

    2015-12-07 08:22:33 <Furai> What I wanted to achieve is /msg <#chan,#chan2,...#chanN> <msg>
    2015-12-07 08:34:40 <FlashCode> Furai: I think the easiest way to solve this is not to add a comma as separator
    2015-12-07 08:34:59 <FlashCode> but to create a specific irc channels list completion for some commands like /msg

**Comment by Mikaela** [comment]
Custom completition for `msg, notice, ctcp` probably works for me too, if those are all the commands needing it.

**Comment by flashcode** [comment]
Yes, and then I have to update `/help notice` and `/help ctcp` to mention that multiple targets are allowed (like in `/msg`).

**Comment by flashcode** [comment]
For `/help ctcp` it was done via #493.
